grakn-engine/src/main/java/ai/grakn/engine/user/SystemKeyspaceUsers.java

```diff
@@ -55,22 +55,22 @@
      */
     @Override
     public boolean addUser(Json userJson) {
-        final Var user = var().isa(USER_ENTITY);
+        final Var[] user = {var().isa(USER_ENTITY)};
```

Comment:
Because Java lambdas are crap! You can't assign to variables in them, but you _can_ assign to the elements of an array, so you can make a single-element array instead.

In a few cases, I've instead replaced the lambda with a `for` loop. We could also replace this with a `for` loop, but I chose not to because we're iterating over a map, which is less readable using a `for` loop (because you have to go `Map.Entry<Blah, Blah>...key...value`).

Ditto for the below comment as well.